index.js

```diff
@@ -105,7 +146,17 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+  let movieFilteredByGenre = [];
```

Comment:
I 😍 this variable name. Might make it plural to be `moviesFilteredByGenre`, since the convention for arrays is plural, but spot on!